core/wiki/macros/lingo.tid

```diff
@@ -15,8 +15,17 @@ $:/language/
 		<!-- First try the old school <active-lingo-base><title> format -->
 		<$transclude $tiddler={{{ [<active-lingo-base>addsuffix<title>] }}} $mode=<<parseMode>>>
 			<!-- If that didn't work, try the new <lingo-base><langcode>/<title> format -->
+			<!-- Get active language, if not language is selected, fallback to English -->
 			<$let language-code={{{  [[$:/language]get[text]get[name]else[en-GB]] }}}>
-        <$transclude $tiddler={{{ [<active-lingo-base>addsuffix<language-code>addsuffix[/]addsuffix<title>] }}} $mode=<<parseMode>>/>
+        <$transclude $tiddler={{{ [<active-lingo-base>addsuffix<language-code>addsuffix[/]addsuffix<title>] }}} $mode=<<parseMode>>>
```

Comment:
Indent level seems to be wrong

Reply:
It is 1 level above let, isn't wrong

Reply:
Sorry. I always put my comments below the issue. So IMO transclude has the wrong indent.

Reply:
<img width="1115" alt="截屏2024-06-19 14 40 54" src="https://github.com/Jermolene/TiddlyWiki5/assets/3746270/e68cfaf6-9910-44ae-b53c-35857ee8d399">

I didn't see any wrong in vscode

Reply:
Have a look at the file. https://github.com/Jermolene/TiddlyWiki5/blob/e9568720489ae5fe9ee74cd16330b8168028beff/core/wiki/macros/lingo.tid#L20C1-L20C9 Line 20 uses spaces instead of tabs for indentation. If you fix that it should work